argo-cd/info.json

```diff
@@ -0,0 +1,894 @@
+{
+  "name": "argo-cd",
+  "version": "1.0.0",
+  "label": "Argo CD",
+  "description": "Argo CD (short for Argo Continuous Delivery) is a declarative, GitOps continuous delivery tool for Kubernetes. It allows you to manage and deploy applications to Kubernetes clusters using Git repositories as the source of truth for both the desired application state and its configuration.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "contributor": "",
+  "help_online": "",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "category": "DevOps and Digital Operations",
```

Comment:
Is this category allowed or are we planning to add this category ??

Reply:
As discuss, provided category is correct.